canonical/ubuntu.com
# /core update images

## Done

- Updated images as per [doc](https://docs.google.com/document/d/1UFpefDRSAcndNBnWp0hDyzqk74aciqhBWJEi05htWmU/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13864.demos.haus/core
- See that the images have been updated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11001
